templates/terraform/encoders/cloud_asset_feed.go.erb

```diff
@@ -0,0 +1,8 @@
+// Remove the "folders/" prefix from the folder ID
```

Comment:
I initially tried to modify the `folder` attribute in the custom encoder. But the automated test failed due to a diff being found (the `folder` parameter was different than expected). So I ended up creating a computed folder_id attribute.
Did I miss something?

Reply:
Sorry! I was likely confusing - I meant adding `folder_id` to the `google_folder` resource schema and then doing a `d.Set()` around [here](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/third_party/terraform/resources/resource_google_folder.go#L121) in the actual resource.
In this way, instead of expecting the user to pass in the name `folders/{{folder_id}}` we'd just expect the `{{folder_id}}`. If we add it to the `google_folder` resource, it will make it easier on the user to get it from the Terraform resource. We should probably add it on the data source as well, but I think the two share some code, so that shouldn't be too bad.
Let me know if that makes sense to you!

Reply:
The way I have implemented it now accepts both folders/12345 and 12345. So I think we are good.
Still, I think it would make sense to add the folder_id attribute to the google_folder resource. But I would not want this to be dependent on that.